tests/tx/test_scripts.py

```diff
@@ -176,22 +176,22 @@ def test_pushdata1(self):
 
     def test_dup(self):
         with self.assertRaises(MissingStackItems):
-            op_dup(ScriptContext(stack=[], logs=[], extras=Mock()))
+            op_dup(ScriptContext(stack=[], logs=[], extras=Mock(), settings=Mock()))
```

Comment:
Simply because this opcode doesn't use the settings, so it doesn't really matter what value is there. I think using a `Mock()` semantically represents that this is not going to be used.

Reply:
Got it. Ok. Thanks!